### ISSUE TITLE
checkbox wizard starts an event to reload the page

### ISSUE THREAD

**Issue by zonky2** [contao/contao#270]
**Affected version(s)**
in Contao ^4.4.21 - also in Contao-Demo 4.6.12

**Description**  
the checkbox wizard starts an event to reload the page

**How to reproduce**  
Test e.g. with the layout settings > CSS files ... move a checkbox...

![screenshot_356](https://user-images.githubusercontent.com/1045318/50984830-0e450400-1503-11e9-922e-fa22b52d07c9.jpg)

**Comment by discordier** [comment]
The problem is, that the "burgers" are `<button>` elements contained within a `<form>`.
Additionally, the event listeners do not prevent the default and as the `<button>` additionally lacks a `type` attribute, it defaults to `type="submit"` and this causes the form to submit.

Suggested change: Add `type="button"` to the burger buttons.
Code location: https://github.com/contao/contao/blob/dda481ec9b3934843302ee8a1353d6cd7cdf9f9b/core-bundle/src/Resources/contao/widgets/CheckBoxWizard.php#L110
```diff
-			$arrOptions[] = $this->generateCheckbox($arrOption, $i, '<button class="drag-handle" title="' . StringUtil::specialchars($GLOBALS['TL_LANG']['MSC']['move']) . '" aria-hidden="true">' . Image::getHtml('drag.svg') . '</button> ');
+			$arrOptions[] = $this->generateCheckbox($arrOption, $i, '<button class="drag-handle" type="button" title="' . StringUtil::specialchars($GLOBALS['TL_LANG']['MSC']['move']) . '" aria-hidden="true">' . Image::getHtml('drag.svg') . '</button> ');

**Comment by leofeyer** [comment]
Fixed in 681f5c43c84787b31c53aca08f743f649905c407.

**Comment by zonky2** [comment]
thx
